live-site/src/hooks/useAirtable.ts

```diff
@@ -10,12 +10,16 @@ interface AirtableData {
 
 export const useAirtableApi = (
   baseName: string,
-  tableName: string
+  tableName: string,
+  isDev = false
 ): AirtableData => {
   const [data, setData] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
   useEffect(() => {
-    const url = `${BASEURL}/${baseName}/${tableName}`;
+    let url = `${BASEURL}/${baseName}/${tableName}`;
```

Comment:
you can do const url = `${BASEURL}/${baseName}/${tableName}${isDev ? "?isDev=true" : ""}`

Reply:
but thats a nit pick, it allows you to make it a const for more safety